### DISCUSSION TITLE
[SUGGESTION] New Obsfucation Method

### DISCUSSION THREAD

**Discussion by Brohammer5** [prometheus-lua/Prometheus#32] [Ideas]
I have seen in some Java obsfucators that create a lot of files. Which require each other, I think those would be a cool feature. And it would make it harder to deobsfucate.

**Comment by levno-710** [comment]
That actually sounds like an interesting idea.
I currently don't have a lot of time to work on prometheus, but I will probably add a feature similar to your suggestion.
